transformers4rec/torch/model/base.py

```diff
@@ -514,6 +518,7 @@ def __init__(
         head_reduction: str = "mean",
         optimizer: Type[torch.optim.Optimizer] = torch.optim.Adam,
         name=None,
+        hf_format=True,
```

Comment:
what does `hf_format` stand for? can we add a short note to the docstring. 

Can we move the docs for the init Parameters down from the class into the init method here.

Reply:
The parameter is specific to NextITemPredictionTask and controls the output format required by the hugging face dependency. This confusing parameter will be removed in #505. In the meantime, I added a short description in the class docstrings.